### ISSUE TITLE
Specify Windows SDK version for `os` dependency

### ISSUE THREAD

**Issue by bdbai** [microsoft/winrt-rs#299]
Currently the `os` dependency pulls in metadata shipped with the system. Imagine some developers that still using Windows 10 1903 (build 18362) due to updates not available. When they want to make use of APIs introduced in build 19041, they would install a new Windows SDK, but there is no way to specify the version to use in winrt-rs.

My suggestion is to extend the syntax for `os` entry in our macros. For example,
```rust
import!(
    dependencies
        os "10.0.19041.0"
        nuget: Microsoft.AI.MachineLearning
    types
        microsoft::ai::machine_learning::*
);
```
Furthermore, we could use semantic versioning to specify a range of versions, like
```rust
import!(
    dependencies
        os "^10.0.19041.0"
        nuget: Microsoft.AI.MachineLearning
    types
        microsoft::ai::machine_learning::*
);
```
In this example, any Windows 10 SDK with build number greater or equals to 19041 would be chosen.

**Comment by kennykerr** [comment]
You can use nuget for that. First specify the SDK version in your toml file like this:

https://github.com/microsoft/winrt-rs/blob/091df31b5bbbb4899d7c656294b0013c860ce5ec/examples/nuget-import/Cargo.toml#L11

And then refer to this nuget package instead of `os` in your import/build macro:

https://github.com/microsoft/winrt-rs/blob/091df31b5bbbb4899d7c656294b0013c860ce5ec/examples/nuget-import/src/main.rs#L3

**Comment by lilianmoraru** [comment]
@kennykerr Is there a way to target Windows 8.1?
Or the metadata the project requires is available only starting with Windows SDK 10.0.17134.1000?

**Comment by kennykerr** [comment]
This describes the default metadata used by the Windows crate: 

https://github.com/microsoft/windows-rs/tree/master/crates/reader/default

You can override this by providing your own winmd files in the `.windows/winmd` folder of your workspace. For example, just pick the SDK you wish to target and copy the winmd files from there. Here's the implementation for reference:

https://github.com/microsoft/windows-rs/blob/70321965518d1d65e07c81a3589226c4910f0d0f/crates/reader/src/workspace.rs#L86-L100

In general I suggest that developers use the latest version and simply stick to the subset of APIs supported by your target OS. You can then support feature light-up more easily on newer versions.
